src/Universum/Debug.hs

```diff
@@ -62,22 +63,27 @@ error s = P.error (unpack s)
 -- | Version of 'Debug.Trace.traceShow' that leaves warning.
 {-# WARNING traceShow "'traceShow' remains in code" #-}
 traceShow :: P.Show a => a -> b -> b
-traceShow a b = trace (P.show a) b
+traceShow a b = trace (pack (P.show a)) b
 
 -- | Version of 'Debug.Trace.traceShow' that leaves warning.
```

Comment:
This comment looks not very accurate, would be good to update while you're at it.

Reply:
Could you formulate comment better then? I don't see why is it bad

Reply:
I mean `s/traceShow/traceShowId`.